tests/unit/machine/Pipeline_unittest.cc

```diff
@@ -50,7 +50,8 @@ TEST_F(PipelineTest, fit_predict)
 	                    ->then(machine);
 
 	// no labels given
-	EXPECT_THROW(pipeline->train(features), ShogunException);
+	//EXPECT_THROW(pipeline->train(features), ShogunException);
+	//pipeline->train(features, labels);
```

Comment:
what happens here?

Reply:
now pipeline have two `train` method, original `train` will not throw a exception.